docs/pages/includes/database-access/redis-connect.mdx

```diff
@@ -0,0 +1,70 @@
+Log into your Teleport cluster and see available databases:
+
+<Tabs>
+  <TabItem scope={["enterprise", "oss"]} label="Self Hosted">
+    ```code
+    $ tsh login --proxy=teleport.example.com --user=alice
+    $ tsh db ls
+    # Name          Description     Labels
+    # ------------- --------------- --------
+    # example-redis Example Redis   env=dev
+    ```
+  </TabItem>
+  <TabItem scope={["cloud"]} label="Cloud">
+    ```code
+    $ tsh login --proxy=mytenant.teleport.sh --user=alice
+    $ tsh db ls
+    # Name          Description     Labels
+    # ------------- --------------- --------
+    # example-redis Example Redis   env=dev
+    ```
+  </TabItem>
+</Tabs>
+
+To connect to a particular database instance, first retrieve its certificate
+using the `tsh db login` command:
+
+```code
+$ tsh db login example-redis
+```
+
+<Admonition type="tip" title="Tip">
```

Comment:
I suggest we remove the `Admonition` tag and make this regular text. I'm not sure how likely it is that a user will miss this after copying the preceding code snippet. Otherwise, this section looks really busy.

Reply:
I can remove it, but we use that tip in may different places and every elsewhere is inside `Admonition`